src/jit/utils.h

```diff
@@ -550,6 +550,60 @@ class AssemblyNamesList2
     }
 };
 
+// MethodSet: Manage a list of methods that is read from a file.
+//
+// Methods are approximately in the format output by JitFunctionTrace, e.g.:
+//
+//     System.CLRConfig:GetBoolValue(ref,byref):bool (MethodHash=3c54d35e)
+//       -- use the MethodHash, not the method name
+//
+//     System.CLRConfig:GetBoolValue(ref,byref):bool
+//       -- use just the name
+//
+// Method names should not have any leading whitespace.
+//
+// TODO: Should this be more related to JitConfigValues::MethodSet?
+//
+class MethodSet
+{
+    // TODO: use a hash table? or two: one on hash value, one on function name
```

Comment:
Maybe we can add a message if `MethodSet ` contains more that 100 methods for example, like: `MethodSet uses a linear search for each method, do not use with long methods list`.

Reply:
I suppose that's possible, at least until using a hash table.